tools/depends/target/Toolchain.cmake.in

```diff
@@ -105,7 +105,8 @@ set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
 set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
 set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 set(CMAKE_FIND_FRAMEWORK LAST)
-set(ENV{PKG_CONFIG_LIBDIR} @prefix@/@deps_dir@/lib/pkgconfig:@prefix@/@deps_dir@/share/pkgconfig)
+# also include native pkgconfig dir to make native binaries used during build available
+set(ENV{PKG_CONFIG_LIBDIR} @prefix@/@deps_dir@/lib/pkgconfig:@prefix@/@deps_dir@/share/pkgconfig:@prefix@/@tool_dir@/lib/pkgconfig:@prefix@/@deps_dir@/share/pkgconfig)
```

Comment:
nooo

Reply:
We need the native binaries though?

Reply:
please define/pass them where needed. If really necessary add $(NATIVEPREFIX)/bin to path, but never globally